python/tvm/relay/backend/graph_executor_codegen.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, mod, target):
         self._get_irmodule = self._mod["get_irmodule"]
         self._setup(mod, target)
 
-    def _setup(self, mod, target):
+    def _setup(self, mod, target: Dict[int, Target]):
```

Comment:
Can you annotate mod too? (Though this kinda seems like a change that accidentally slipped into this PR).